pom.xml

```diff
@@ -33,6 +33,11 @@
 			<artifactId>h2</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<scope>runtime</scope>
```

Comment:
why you added this dependancy. you no need this now.

Reply:
then how we connect a database

Reply:
Yes, but for the now we dont have db right.

Reply:
Can you correct this?

Reply:
done